### DISCUSSION TITLE
5-inch screen - frame time graph

### DISCUSSION THREAD

**Discussion by Rembo777** [mathoudebine/turing-smart-screen-python#613] [Q&A]
Good day, esteemed gentlemen. Could anyone advise if it is possible to display a frame time graph on a 5-inch screen? I would like to place it in the spot currently occupied by hard drive and network sensors. At the moment, I am using the Turzx program, but it doesn't have a graph feature. I've seen something similar in your threads, but it was for a 3.5-inch screen.
![Снимок экрана 2024-12-08 200359](https://github.com/user-attachments/assets/c2bbdc24-da36-43bd-9541-6834559bcd1b)
![2024-12-08 20-05-32](https://github.com/user-attachments/assets/5c6643d9-c2e1-4b29-86cd-1bd63b834c28)

**Comment by c0ldJS** [comment]
Line graphs that show changes over time are available on all panel sizes. The only difference between the 3½" and 5" (or any of the others) is how much screen real estate you have to play with. That's all.

Some of the themes available in the project today use line graphs on 5" displays.

I use a 5" screen and my theme has two line graphs that show changes in CPU and GPU temperatures.
